docs-react-native/react-native/docs/ios/remote-notification-support.md

```diff
@@ -68,7 +68,17 @@ At this point everything should still be running normally. This is the final ste
 Before, moving to the next step, run the app and check it builds successfully – make sure you have the correct target selected. 
 
 ## Edit the payload
-Now everything is setup in your app, you can alter your notification payload to include `notifee_options`:
+Now everything is setup in your app, you can alter your notification payload in two ways:
+
+1. When sending from your BE (either via REST api or firebase admin sdk)
+2. In Notification Service Extension when you receive notification
```

Comment:
I have difficulty troubleshooting sometimes because messages and notifications etc are used interchangeably when they actually are specific and different, so I'm just being pedantic here by being specific with "remote message" but it may help future troubleshooting

```suggestion
2. In a Notification Service Extension in your app when a device receives a remote message
```